### ISSUE TITLE
--headers is not working

### ISSUE THREAD

**Issue by George-Miao** [ducaale/xh#222]
Trying to use xh to debug my web server. I use this command:

```bash
xh localhost:8080 -p h

# and 

xh localhost:8080 --headers
```

Both should only print the header, however I can still see the body being printed.

```
HTTP/1.1 200 OK
Content-Length: 2
Content-Type: text/plain; charset=utf-8
Date: Mon, 17 Jan 2022 00:13:05 GMT

OK
``` 

Am I missing something? Since I use the same argument as httpie.

**Comment by blyxxyz** [comment]
This was a bug, but it should be fixed in version 0.14.1. Are you perhaps running 0.14.0? (You can check with `xh --version`.)

**Comment by George-Miao** [comment]
> xh --version

Ah right. Looks like I should update my package first. Thanks for your time.
